sympy/core/expr.py

```diff
@@ -1957,11 +1957,15 @@ def as_powers_dict(self):
         d.update(dict([self.as_base_exp()]))
         return d
 
-    def as_coefficients_dict(self):
+    def as_coefficients_dict(self, *syms):
         """Return a dictionary mapping terms to their Rational coefficient.
-        Since the dictionary is a defaultdict, inquiries about terms which
-        were not present will return a coefficient of 0. If an expression is
-        not an Add it is considered to have a single term.
+            Since the dictionary is a defaultdict, inquiries about terms which
+            were not present will return a coefficient of 0.
```

Comment:
```suggestion
        Since the dictionary is a defaultdict, inquiries about terms which
        were not present will return a coefficient of 0.
```